multiply_ui/server/controllers/controller.py

```diff
@@ -3,22 +3,6 @@
 
 class Controller(object):
 
-    __instance = None
-
-    @staticmethod
-    def get_instance():
-        """ Static access method. """
-        if Controller.__instance is None:
-            Controller()
-        return Controller.__instance
-
-    def __init__(self):
-        """ Virtually private constructor. """
-        if Controller.__instance is not None:
-            raise Exception("This class is a singleton!")
-        else:
-            Controller.__instance = self
-
     def get_parameters(self):
         with open('test/test_data/example_parameters.json') as f:
```

Comment:
Path changed, see my screenshot in Skype!
Don't use file paths, they wont work after packaging, instead do
```
import pkg_resources
json_text = pkg_resources.resource_string("multiply_ui", "server/resources/processing-parameters.json")
return json.loads(json_text)
```